html5lib/html5parser.py

```diff
@@ -901,7 +901,7 @@ def __init__(self, parser, tree):
                 ("iframe", self.startTagIFrame),
                 (("noembed", "noframes", "noscript"), self.startTagRawtext),
                 ("select", self.startTagSelect),
-                (("rp", "rt"), self.startTagRpRt),
+                (("rp", "rt", "rb", "rtc"), self.startTagRpRtRbRtc),
```

Comment:
I'm not sure that this PR is up to date with the more recent changes to the spec anyway.